framework/src/main/java/org/checkerframework/common/wholeprograminference/WholeProgramInferenceImplementation.java

```diff
@@ -841,6 +841,10 @@ private void updateAtmWithLub(AnnotatedTypeMirror sourceCodeATM, AnnotatedTypeMi
 
     switch (sourceCodeATM.getKind()) {
       case TYPEVAR:
+        if (!(ajavaATM instanceof AnnotatedTypeVariable)) {
```

Comment:
This change looks unrelated to WPI?

Reply:
It is not - this fixes the crashes I was talking about on our call earlier. (This is the code that merges multiple estimates within WPI.)

Reply:
I guess it is only exposed with current tests once inference under warning suppression is enabled?

Reply:
That's correct.